ballerina-shell/modules/shell-cli/src/main/resources/command.header.txt

```diff
@@ -0,0 +1,9 @@
+  ____        _ _           _                _____ _          _ _
+ |  _ \      | | |         (_)              / ____| |        | | |
+ | |_) | __ _| | | ___ _ __ _ _ __   __ _  | (___ | |__   ___| | |
+ |  _ < / _` | | |/ _ | '__| | '_ \ / _` |  \___ \| '_ \ / _ | | |
+ | |_) | (_| | | |  __| |  | | | | | (_| |  ____) | | | |  __| | |
+ |____/ \__,_|_|_|\___|_|  |_|_| |_|\__,_| |_____/|_| |_|\___|_|_|
+=============================================== REPL for Ballerina
+Welcome to Ballerina Shell REPL.
+Type /exit to exit and /help to get about information.
```

Comment:
How about `Type /exit to exit and /help for help` ?

Reply:
Or list of available commands ?

Reply:
Btw, @kdsuneraavinash should remove the BIG Ballerina logo at the start of the shell?

Reply:
Ok I'll remove everything except for 
```
Welcome to Ballerina Shell REPL.
Type /exit to exit and /help to get about information.
```

or should I remove the welcome as well?